tests/SprykerSdkTest/Spryk/Integration/Client/Dependency/Client/AddClientDependencyClientBridgeInterfaceMethodTest.php

```diff
@@ -31,6 +31,8 @@ class AddClientDependencyClientBridgeInterfaceMethodTest extends Unit
     protected $tester;
 
     /**
+     * @skip
```

Comment:
Can we add a reason why? So in the future we know?

Reply:
This is the only group of skipped tests and they all will be processed in scope of particular ticket. I see no reason to leave detailed information here, rather in ticket. As for the skip message in general - I think some general message about intention to unskip tests in future should be enough.